examples/dmd/wave_equation.cpp

```diff
@@ -487,8 +548,15 @@ int main(int argc, char *argv[])
 
             if (!last_step) {
                 curr_window++;
-                dmd_u.push_back(new CAROM::DMD(u.Size(), dt));
-                dmd_u[curr_window]->takeSample(u.GetData(), t);
+                if(snapshotDMD)
+                {
+                    dmd_u.push_back(new CAROM::SnapshotDMD(u.Size(), dt));
+                }
+                else
+                {
+                    dmd_u.push_back(new CAROM::DMD(u.Size(), dt));
+                }
+                dmd_u[curr_window]->takeSample(u, t);
```

Comment:
`u.GetData()` (to avoid deprecated warning)

Reply:
fixed